src/Take.Blip.Builder/FlowLoader.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Take.Blip.Builder.Models;
+
+namespace Take.Blip.Builder
+{
+    internal class FlowLoader : IFlowLoader
+    {
+        public Task<Flow> LoadFlowAsync(FlowType flowType, Flow parentFlow, string identifier, CancellationToken cancellationToken)
+        {
+            //Used when occurs a subflow redirect.Wille be implemented in the future.
```

Comment:
Suggestion, if I understood correctly what you meant.

`This should be called when redirecting to a subflow. This feature is not supported yet, so we throw an exception.